torq.q

```diff
@@ -201,7 +201,7 @@ pubmap:@[value;`pubmap;`ERROR`ERR`INF`WARN!1 1 0 1]
 
 // Log a message
 l:{[loglevel;proctype;proc;id;message;dict]
-	$[0 < redir:`int$(0w 1 `onelog in key .proc.params)&0^outmap[loglevel];
+	if[0 < redir:`int$(0w 1 `onelog in key .proc.params)&0^outmap[loglevel];
 		neg[redir] .lg.format[loglevel;proctype;proc;id;message];
 		ext[loglevel;proctype;proc;id;message;dict]];
```

Comment:
I think you want to call the ext function in either case - if 0<redir or not. So can you change that, and fix up the indenting

Reply:
so if 0<redir we want to run both of the above, if not, we just want to run the extended logging?